makePDF.py

```diff
@@ -184,6 +184,24 @@ def run ( self ):
 				content.extend(warnings)
 			else:
 				content.append("")
+
+			s = sublime.load_settings("LaTeXTools.sublime-settings")

```

Comment:
ST2 raises an exception when you try to load settings [or do basically anything with the Sublime API off the main thread](http://stackoverflow.com/questions/29975674/which-operations-in-the-sublime-text-2-api-are-thread-safe). This setting should be loaded in the `run()` function of the `make_pdfCommand` class where the other settings are loaded and then accessed something like `self.caller.hide_panel_level`.